docs/en/connector-v2/source/kafka.md

```diff
@@ -71,6 +71,21 @@ Source plugin common parameters, please refer to [Source Common Options](common-
 
 The structure of the data, including field names and field types.
 
+for example:
+
+```hocon
+   schema = {
+      fields {
+        name = "string"
+        age = "int"
+      }
+      meta.fields {
```

Comment:
should we support meta in schema field? I think we need unified design, not only used by kafka. @TyrantLucifer @ashulin @hailin0

Reply:
Agree with you. IMO, these datas of meta also a part of one record. Schema configuration should be used to resolve real received data. When the data being converted to SeaTunnelRow, connector should merge the metadata and upstream data before parsing it through Shema.

Reply:
Meta data is required for both Source and Sink connector.Support meta in schema field is more general for connectors.

Reply:
> Meta data is required for both Source and Sink connector.Support meta in schema field is more general for connectors.

Yes, should we should parse meta info in SeaTunnelSchema, not in kafka connector. Before do this, should design general meta field how to work.

Reply:
> > Meta data is required for both Source and Sink connector.Support meta in schema field is more general for connectors.
> 
> Yes, should we should parse meta info in SeaTunnelSchema, not in kafka connector. Before do this, should design general meta field how to work.

Can we define relevant interfaces for metadata in the schema and implement each connector accordingly, because the metadata of each connector is different and incompatible

Reply:
> > > Meta data is required for both Source and Sink connector.Support meta in schema field is more general for connectors.
> > 
> > 
> > Yes, should we should parse meta info in SeaTunnelSchema, not in kafka connector. Before do this, should design general meta field how to work.
> 
> Can we define relevant interfaces for metadata in the schema and implement each connector accordingly, because the metadata of each connector is different and incompatible

This is a good idea, can you implement it with create an new issue and PR. Thanks!

Reply:
@Hisoka-X Ok, I'll try